### ISSUE TITLE
Terminal doesn't recognize `/` correctly in vi-command mode

### ISSUE THREAD

**Issue by timonson** [alacritty/alacritty#4851]
### System

OS: Linux/s
Version: alacritty 0.7.2 (5ac8060b)
Linux/BSD: X11, i3

### Description

I put the following code into my .inputrc using Bash:

```bash
set keymap vi-command
"ciw": "lbcw"
```

Now, when I have a string which contains a slash, e.g. `aaa/bbb`, and press the `ciw` key combination very fast while the cursor is over a letter `b`, the cursor deletes the string `bbb` but jumps before the `/` (what should not happen). 
It is strange that this *bug* does **not** happen when I press the keys slowly. It also does not happen with another terminal, e.g. `st`.

Thanks!

**Comment by chrisduerr** [comment]
That sounds like you're talking about an issue with bash's vi mode?

**Comment by timonson** [comment]
Yes

**Comment by chrisduerr** [comment]
Why do you think this has anything to do with Alacritty? It has nothing to do with bash's vi mode.

**Comment by timonson** [comment]
Because this wrong behavior doesn't appear with other terminals. Additionally, I know that my mentioned settings in my inputrc are correct. I don't know for sure it is caused by Alacritty but I have to start somewhere, right? What do you think is causing this?

**Comment by chrisduerr** [comment]
You said yourself this isn't happening when you press your keys slower, that could easily explain the difference between terminals.

How many other terminals have you tried and which TERMs did you test?

**Comment by timonson** [comment]
I made a mistake in my description. The wrong behavior always happens with `ciw` (no matter how fast I press the keys) but it does not happen when I press the original key combination `lbcw` **slowly**. I tested it only with `st` before but now I am not able to replicate a different behavior compared to Alacritty anymore. Something different is apparently causing this - like you mentioned. Thanks for the help and sorry for the confusion!
